README.md

```diff
@@ -0,0 +1,18 @@
+# Noboty Practice Bot Code
+
+> Code for the 2018 practice bot
+
+## Noboty?
+Noboty is a play on words from Team 118's Everybot. We called it this because noboty will use it.
+
+## Controls
+We used a one-joystick control system for sake of convenience. Moving the joystick on the Y axis moves the robot forwards or backwards, and the X axis causes it to rotate.
```

Comment:
Also explain how we account for the joystick being weird. ((0,0) is the bottom right and we had to add some corrections.)